corehq/apps/registry/static/registry/js/registry_edit.js

```diff
@@ -13,6 +13,8 @@ hqDefine("registry/js/registry_edit", [
     'hqwebapp/js/bootstrap5/knockout_bindings.ko', // openModal
     'hqwebapp/js/bootstrap5/main', // makeHqHelp
     'hqwebapp/js/multiselect_utils',
+    'hqwebapp/js/components/inline_edit',
+    'hqwebapp/js/components/pagination',
```

Comment:
I don't know JS and am curious why these need to be included in here when components.ko wasn't included before?